### PR TITLE
Add workflow image to project overview page [fixes SCI-41]

### PR DESCRIPTION
**Pull request by ZmagoD**
https://biosistemika.atlassian.net/browse/SCI-41

### CODE REVIEW

File: app/models/experiment.rb [hunk 1]
```diff
@@ -195,6 +200,109 @@ def update_canvas(
...
+                                      label: label,
+                                      pos: "#{child_mod.x},-#{child_mod.y}!")
+              graph.add_edges(child_node, parent)
+            end
+          end
+        end
+      end
+    end
+
+    file_location = "/tmp/workflowimg_#{Process.pid}.png"
```
Comment (jbargu) [inline app/models/experiment.rb]: I'd maybe rather use temp file, which guarantees unique file:

` file = Tempfile.open("img", Rails.root.join("tmp"))`

**Comment by ZmagoD** [comment]
Uff I forgot to copy it :smile:

**Comment by jbargu** [comment]
No worries, we're here for you. :kissing_smiling_eyes:

**Comment by jbargu** [comment]
I've tested the code on production environment - with some tweaks it now works.

I reviewed the MR. Looks good :+1:! You won't escape employee of the week for sure.
:shipit:

**Pull request merged**